fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContextConfig.java

```diff
@@ -597,6 +628,39 @@ public Builder setRecordContextProperties(@Nonnull final RecordLayerPropertyStor
             return this;
         }
 
+        /**
+         * Get tags used for throttling.
+         * @return throttling tags
+         */
+        @Nonnull
+        public Set<String> getTags() {
+            return tags;
+        }
+
+        /**
+         * Set tags used for throttling.
+         * @param tags new set of tags
+         */
+        public void setTags(@Nonnull final Set<String> tags) {
+            this.tags = tags;
+        }
+
+        /**
+         * Get whether to report conflicting key ranges on commit conflict.
+         * @return {@code true} to report conflicting keys
+         */
+        public boolean isReportConflictingKeys() {
+            return reportConflictingKeys;
+        }
+
+        /**
+         * Set whether to report conflicting key rangs on commit conflict.
+         * @param reportConflictingKeys {@code true} to report conflicting keys
+         */
+        public void setReportConflictingKeys(final boolean reportConflictingKeys) {
```

Comment:
Should these two new setters return `Builder`s to make them more fluent?

Reply:
Yes, they should. Fixed.